cpu/stm32f4/include/cpu_conf.h

```diff
@@ -50,7 +50,11 @@ extern "C" {
  * @{
  */
 #define CPU_DEFAULT_IRQ_PRIO            (1U)
+#if defined(CPU_MODEL_STM32F413ZH) || defined(CPU_MODEL_STM32F413VG)
+#define CPU_IRQ_NUMOF                   (102U)
```

Comment:
Fixed. But it seems that for other part numbers, some vector definitions are wrong.